disperser/Makefile

```diff
@@ -63,4 +66,32 @@ run_encoder: build_encoder
   --disperser-encoder.log.level-file debug
 
 run_dataapi_tests:
-	go test -v disperser/dataapi
\ No newline at end of file
+	go test -v disperser/dataapi
+
+run_blobapi: build_blobapi
+	./bin/blobapi \
+	--disperser-server.s3-bucket-name myDisperserBucket \
+	--disperser-server.dynamodb-table-name myDisperserTable \
+	--disperser-server.grpc-port 51002 \
+	--disperser-server.bls-operator-state-retriever 0x9d4454B023096f34B160D6B654540c56A1F81688 \
+	--disperser-server.eigenda-service-manager 0x67d269191c92Caf3cD7723F116c85e6E9bf55933 \
+	--disperser-server.enable-metrics=false \
+	--chain.rpc http://localhost:8545 \
+	--chain.private-key ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
```

Comment:
is this a throw away key?
i think you can also use blank key value like `--chain.private-key ''\`